config/plugins/visualizations/charts/static/repository/visualizations/ngl/viewer/wrapper.js

```diff
@@ -0,0 +1,47 @@
+define( [ 'utilities/utils', "plugins/ngl/viewer" ], function( Utils, ngl ) {
+    return Backbone.Model.extend({
+        initialize: function( options ) {
+            var dataset = options.dataset,
+                settings = options.chart.settings,
+                url = window.location.protocol + '//' + window.location.host + "/datasets/" + dataset.dataset_id +
```

Comment:
I think this should be `url = dataset.download_url`. Then it will also work if the instance url is proxied.